### ISSUE TITLE
First connection alert stays even when you've logged in.

### ISSUE THREAD

**Issue by lhark** [rezometz/paiji2#40]
This alert is nearly taking all my screen on my laptop and is now useless to me or other users that have claimed their account.

**Comment by liusiqi43** [comment]
We should allow alert to have conditions then.

**Comment by gpichot** [comment]
Yes. I think it should be done in the same way that the management of
permissions. But we have to discuss about it first (tricky one). I will be
in Metz between the 24th Jan and 2nd Feb, we can schedule a meeting during
this period.

2015-01-14 15:57 GMT+08:00 Siqi LIU <notifications@github.com>:

> We should allow alert to have conditions then.
>
